demos/drawer/permanent-drawer-below-toolbar.html

```diff
@@ -135,6 +152,16 @@ <h1 class="mdc-typography--display1">Permanent Drawer</h1>
       document.querySelector('#toggle-tall').addEventListener('click', function() {
         extraTall.style.display = extraTall.style.display ? '' : 'none';
       });
+
+      var drawerEl = document.querySelector('.mdc-drawer');
```

Comment:
For future-proofing, I would probably give both these elements an `id` and select on that.

E.g., `id="demo-drawer"` and `id="demo-theme-switch"`

Ditto for other files